docker/cygnus-ngsi/Dockerfile

```diff
@@ -65,9 +67,9 @@ RUN yum -y install tar nc which git java-1.6.0-openjdk-devel && \
 # Create Cygnus log folder
 RUN mkdir ${FLUME_HOME}/log
 
-# Clone fiware-cygnus
-RUN git clone ${GIT_URL_CYGNUS} ${CYGNUS_HOME} && \
-    cd ${CYGNUS_HOME} && \
+# Instead of re-cloning, copy the fiware-cygnus repo, move there and checkout the propper branch
+COPY . ${CYGNUS_HOME}
```

Comment:
I don't see any benefit, specially if you already have to download and install the dependencies for building cygnus, as they will take more time to download than the clone.  Also, using a shallow clone, as pointed by @albertinisg, will clone just the last revision instead of the full repository.